ledger_device_sdk/src/ui/gadgets.rs

```diff
@@ -66,6 +66,51 @@ pub fn clear_screen() {
     BLANK.paint();
 }
 
+/// Display a developer mode / pending review popup, cleared with user interaction.
+///
+/// This method must be called by an application at the very beginning until it has been reviewed
+/// and approved by Ledger.
+///
+/// # Arguments
+///
+/// * `comm` - Communication manager used to get device events.
+///
+/// # Examples
+///
+/// Following is an application example main function calling the pending review popup at the very
+/// beginning, before doing any other application logic.
+///
+/// ```
+/// #[no_mangle]
+/// extern "C" fn sample_main() {
+///     let mut comm = Comm::new();
+///     ledger_device_sdk::ui::gadgets::display_pending_review(&mut comm);
+///     ...
+/// }
+/// `
+pub fn display_pending_review(comm: &mut Comm) {
+    clear_screen();
+
+    // Add icon and text to match the C SDK equivalent.
+    if cfg!(target_os = "nanos") {
+        "Pending".place(Location::Custom(2), Layout::Centered, true);
+        "Ledger review".place(Location::Custom(14), Layout::Centered, true);
```

Comment:
Something like
```
[ "Pending", "Ledger review" ].place(Location::Middle, Layout::Centered, true);
```
should work also?

Reply:
I doubt this would make it pixel perfect with the C SDK version, and that's what I wanted.